sonaranalyzer-dotnet/src/SonarAnalyzer.CSharp/Rules/CallToAsyncMethodShouldNotBeBlocking.cs

```diff
@@ -96,9 +98,10 @@ private static void ReportOnViolation(SyntaxNodeAnalysisContext context)
                 return;
             }
 
-            if (simpleMemberAccess.IsInNameofCall(context.SemanticModel))
+            if (IsResultInContinueWithCall(memberAccessNameName, simpleMemberAccess) ||
```

Comment:
Can we move this above `possibleMemberAccesses` to gain little performance improvement? `GetSymbolInfo` is more expensive.